src/plugins/intel_gna/ops/copy.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ov {
+namespace intel_gna {
+namespace op {
+/// \brief GNA specific copy layer operation
+///
+class Copy : public ngraph::op::Op {
```

Comment:
> Could you explain a bit how this ngraph Op is converted to Legacy (CNN) representation ?

For operations without weights convertion is doing automatically. Otherwise an adapter is needed in legacy code.